### PR TITLE
LPD-35176 Enable edit object entry through workflow task page

### PR DESCRIPTION
**Pull request by p-albuquerque**
Related issue: https://issues.liferay.com/browse/LPD-35176

### CODE REVIEW

**Comment by p-albuquerque** [comment]
ci:test:sf

**Review by marcelabc — APPROVED** [review]
LGTM 🎉
the backend code.

We need to ask a frontend review for the test :+1:

**Comment by marcelabc** [comment]
Hey @rebsilva 
Could you help and review this PR, please? :smile:

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
ci:test:playwright

File: modules/test/playwright/pages/portal-workflow-task-web/WorkflowTaskDetailsPage.ts [hunk 1]
```diff
@@ -57,6 +58,7 @@ export class WorkflowTaskDetailsPage {
 		this.reviewActionMenu = page.locator(
 			'[id="_com_liferay_portal_workflow_task_web_portlet_MyWorkflowTaskPortlet_kldx___menu"]'
 		);
+		this.editAssetButton = page.locator('[title="Edit"]');
```
Comment (rebsilva) [inline modules/test/playwright/pages/portal-workflow-task-web/WorkflowTaskDetailsPage.ts:61]: can you please order this? it's bellow the doneButton like you did when declaring the locator itself

**Review by rebsilva — CHANGES_REQUESTED** [review]
hi @p-albuquerque, great work here! i just asked for a minor change :cherry_blossom:

**Comment by p-albuquerque** [comment]
ci:test:sf

**Review by rebsilva — APPROVED** [review]
LGTM! :tada:

**Comment by rebsilva** [comment]
@p-albuquerque the code is approved, but i realized that the test you've created is failing on the ci; could you please take a look?

`Manage object entries through Workflow › can edit object entry through workflow task page: objectEntries.spec.ts:612:2 can edit object entry through workflow task page`

**Comment by p-albuquerque** [comment]
ci:test:playwright

**Comment by p-albuquerque** [comment]
ci:stop

**Comment by p-albuquerque** [comment]
ci:test:playwright

**Comment by p-albuquerque** [comment]
ci:test:playwright

**Comment by p-albuquerque** [comment]
ci:report:79319055

**Comment by p-albuquerque** [comment]
ci:test:playwright

**Comment by marcelabc** [comment]
This is the Jenkins report related to object-web/objectEntries.spec:
https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/973035/testReport/object-web_objectEntries.spec/ts/

**Comment by marcelabc** [comment]
It would be better for us to analyze this section **"Failures unique to this pull:"** of the reports generated by ci:test:playwright.
Screenshot from 2024-09-04 12-24-39
Specially the part regarding `object-web`

**Comment by p-albuquerque** [comment]
@marcelabc I will check this. Thanks!

**Comment by p-albuquerque** [comment]
ci:forward

**Pull request closed without merging**